### PR TITLE
Add API docs for authorization endpoints

### PR DESCRIPTION
**Pull request by anu-rock**
#### Changes proposed in this Pull Request

Add API documentation for authorization endpoints - get all authorizations, get a single authorization, and get summary of all authorizations.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (does not apply)
- [ ] Tested on mobile (does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

File: docs/rest-api/source/includes/wp-api-v3/authorization.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+	"outcome_type": "authorized",
+	"status": "succeeded"
+}
+```
+
+## Get authorization summary
+
+_@since v4.9.0_
+
+Return a high-level summary of all authorization data.
```
Comment (elazzabi) [inline docs/rest-api/source/includes/wp-api-v3/authorization.md]: In the `/authorizations` docs, you mentioned: **active** authorizations, and I think you meant those you can still capture (ie, less than 7 days old).

I think we should do the same here, given the summary is only for **active** authorizations.

**Review by elazzabi — CHANGES_REQUESTED** [review]
Looks good! I only left one change request to make it coherent with everything else.

File: docs/rest-api/source/includes/wp-api-v3/authorization.md (hunk 1 above)
Comment (anu-rock) [reply docs/rest-api/source/includes/wp-api-v3/authorization.md]: Great point! Fixed in 0156100. What do you think?

Comment (elazzabi) [reply docs/rest-api/source/includes/wp-api-v3/authorization.md]: Looks good now, thanks!

**Review by elazzabi — APPROVED** [review]

**Pull request merged**